go/obscuronode/host/p2p/local_p2p.go

```diff
@@ -0,0 +1,124 @@
+package p2p
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/obscuronet/obscuro-playground/go/log"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/nodecommon"
+)
+
+// fake multicast network
```

Comment:
Can this comment remind the reader that this is shared across all `LocalP2P` instances? Caught me out.